lib/node_modules/@stdlib/string/len/lib/is-surrogate.js

```diff
@@ -0,0 +1,55 @@
+/**
```

Comment:
For file names, prefer using underscores, rather than hyphens. For example, here, `is_surrogate`. Applies here and elsewhere.